### ISSUE TITLE
I can't restore my Teslalogger.

### ISSUE THREAD

**Issue by Skamaica** [bassmaster187/TeslaLogger#1341]
Hello :)

Teslalogger runs in Docker for me

I selected the Mysqldump file as described in the instructions for restoring. After about 10 minutes the following message appears: filename: Size compressed

I also unpacked the file but then a white window appears after a short time.

There is nothing in the log.

**Comment by bassmaster187** [comment]
Docker or raspberry 3 / 4?

**Comment by Skamaica** [comment]
Docker

**Comment by Skamaica** [comment]
I have now copied the entire TeslaLogger folder. This works now but Grafana doesn't start. I ran chmod 777 TeslaLogger/GrafanaDB and also updated Compose.yaml according to the instructions. Message remains and Grafana does not start

**Comment by Skamaica** [comment]
the Grafana container always restarts. I have already deleted it and re-pulled it but no change
<img width="1783" alt="Bildschirmfoto 2024-07-26 um 09 10 56" src="https://github.com/user-attachments/assets/45135473-9dbe-427f-a2d1-4272ac7633cf">

**Comment by michib29** [comment]
I have the same problem. I tried to restore a backup. Once from a Raspberry Pi 3 to a Docker instance and vice versa, both times the same message appeared in the browser: filename: Size compressed:

**Comment by superfloh247** [comment]
What size is your backup.gz file?

**Comment by michib29** [comment]
> What size is your backup.gz file?

131.8 MB

**Comment by superfloh247** [comment]
This might be above some Apache and/or php limits, I know I had issues there and restored manually with dbeaver.
I‘ll try to check.

**Comment by michib29** [comment]
How can I perform the restore with Dbeaver?

**Comment by Skamaica** [comment]
> How can I perform the restore with Dbeaver?

I would be interested in that too

**Comment by superfloh247** [comment]
1. unpack mysqldump20240820010001.gz to mysqldump20240820010001 and add .sql extension mysqldump20240820010001.sql
2. give dBeaver a lot of ram https://github.com/dbeaver/dbeaver/wiki/Command-Line#vm-arguments
3. open SQL script for your TeslaLogger connection
4. load mysqldump20240820010001.sql and execute

**Comment by michib29** [comment]
is possible to insert the DB dump from an old Docker instance into the latest Docker version of Telsalogger using DBbeaver

**Comment by bassmaster187** [comment]
> is possible to insert the DB dump from an old Docker instance into the latest Docker version of Telsalogger using DBbeaver

Yes, make sure you expose the database ports to make it accesible. Just uncomment the line and start docker.

https://github.com/bassmaster187/TeslaLogger/blob/bfa3b8c4259db36ccc582615df57f5cc618475b1/docker-compose.yml#L48

**Comment by michib29** [comment]
Thank you. That worked.
